EMS/core-bundle/src/Command/Submission/ExportCommand.php

```diff
@@ -38,28 +40,47 @@ public function __construct(
     #[\Override]
     protected function configure(): void
     {
-        $this->addArgument(self::ARGUMENT_CONFIG_FILE, InputArgument::REQUIRED, 'JSON config file (path)');
+        $this->addArgument(self::ARGUMENT_CONFIG, InputArgument::REQUIRED, 'JSON config file (path or JSON)');
     }
 
     #[\Override]
     protected function initialize(InputInterface $input, OutputInterface $output): void
     {
         parent::initialize($input, $output);
 
-        $fileIdentifier = $this->getArgumentString(self::ARGUMENT_CONFIG_FILE);
-        $file = $this->getFile($fileIdentifier);
-
-        $this->exportConfig = ExportConfig::fromJson($file->getContent());
+        $this->exportConfig = $this->getExportConfig();
     }
 
     #[\Override]
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $this->exporter->export($this->exportConfig, $this->io);
+        $this->io->section('Exporting form submissions');
+
+        $result = $this->exporter->export($this->exportConfig);
+
+        if (0 === $result->exportCount) {
+            $this->io->warning(\sprintf('No exported submissions on %d unprocessed submissions. No file or emails were generated.', $result->unprocessedSubmissionsCount));
+
+            return self::EXECUTE_SUCCESS;
+        }
+
+        $this->io->success(\sprintf('Exported %d submissions on %d unprocessed submissions.', $result->exportCount, $result->unprocessedSubmissionsCount));
 
         return self::EXECUTE_SUCCESS;
     }
 
+    private function getExportConfig(): ExportConfig
+    {
+        $input = $this->getArgumentString(self::ARGUMENT_CONFIG);
+
+        $config = match (true) {
+            Json::isJson($input) => $input,
+            $this->getFile($input) instanceof FileInterface => $this->getFile($input)->getContent(),
+        };
+
+        return ExportConfig::fromJson($config);
+    }
+
```

Comment:
May I suggest to refactor this into a $this->storageManager->getConfig($fileIdentifier). This getConfig method should be able to  test id the provided string is a JSON, a real path to a file or a hash within a storage service